tasks/gravity.yml

```diff
@@ -8,11 +8,19 @@
       command: "{{ galaxy_venv_dir }}/bin/galaxyctl register {{ galaxy_config_file }}"
       args:
         creates: "{{ galaxy_gravity_state_dir }}/configstate.yaml"
+      when: __galaxy_major_version is version('23.0', '<')
 
     - name: Update Gravity process management files
       command: "{{ galaxy_venv_dir }}/bin/galaxyctl update"
       args:
         creates: "{{ galaxy_gravity_state_dir }}/supervisor/supervisord.conf.d"
+      when: __galaxy_major_version is version('23.0', '<')
+
+    - name: Update Gravity process management files
+      command: "{{ galaxy_venv_dir }}/bin/galaxyctl -c {{ galaxy_config_file }} update"
+      args:
+        creates: "{{ galaxy_gravity_state_dir }}/supervisor/supervisord.conf.d"
```

Comment:
this still looks supervisord specific, not sure if we need to do something else here under systemd?

Reply:
Should all be stateless at this point, I think we can just delete this

Reply:
actually i think the `galaxy_systemd_mode == gravity` case is still the default using supervisor and all, so this is probably correct ?

Reply:
Do we need a separate `galaxy_systemd_mode == gravity-systemd` then or similar?

Judging by https://github.com/galaxyproject/ansible-galaxy/blob/main/defaults/main.yml#L383 there just isn't support for the systemd version of gravity in here yet and that might need more changes (unless i'm misreading)

Reply:
It just depends on your Gravity config, there shouldn't be anything pm-specific.

Yes it's stateless now but supervisor mode still has to create supervisor PMs and those go in the "state dir" (which is now inside Galaxy's `data_dir` unless you explicitly set the Gravity state dir).